openreview/conference/helpers.py

```diff
@@ -246,11 +249,13 @@ def get_decision_stage(client, request_forum):
             due_date = decision_due_date,
             public = request_forum.content.get('make_decisions_public', '').startswith('Yes'),
             release_to_authors = request_forum.content.get('release_decisions_to_authors', '').startswith('Yes'),
-            release_to_reviewers = request_forum.content.get('release_decisions_to_reviewers', '').startswith('Yes'))
+            release_to_reviewers = request_forum.content.get('release_decisions_to_reviewers', '').startswith('Yes'),
+            email_authors = True)
```

Comment:
One question: not sure where we use get_decision_stage, but why do we set email_authors=True here?

Reply:
get_decision_stage is being executed in the process function of 'Decision Stage' of the request form. 

I put True because I think in most of the cases where the venue is managed by the PCs they want to notify the authors. But we could add a checkbox in the Decision Stage form, what do you think?

Reply:
Oh I see. I think that makes sense, you are right most PCs do assume that when decisions are posted authors are emailed about it. Adding the checkbox wouldn't hurt.

Reply:
Ok, I implemented the option in the ui of the decision stage, please review this PR too: https://github.com/openreview/openreview-scripts/pull/1181

Reply:
Looks good!